resources/hpxml-measures/BuildResidentialHPXML/measure.rb

```diff
@@ -3603,10 +3603,10 @@ def self.get_adiabatic_adjacent_surface(model, surface)
       next if surface == adjacent_surface
       next if adjacent_surface.surfaceType != adjacentSurfaceType
       next if adjacent_surface.outsideBoundaryCondition != 'Adiabatic'
-      next if Geometry.getSurfaceXValues([surface]) != Geometry.getSurfaceXValues([adjacent_surface])
-      next if Geometry.getSurfaceYValues([surface]) != Geometry.getSurfaceYValues([adjacent_surface])
-      next if Geometry.getSurfaceZValues([surface]) != Geometry.getSurfaceZValues([adjacent_surface])
-
+      next if Geometry.getSurfaceXValues([surface]).sort != Geometry.getSurfaceXValues([adjacent_surface]).sort
```

Comment:
@aspeake Probably want to make this change on OS-HPXML, and then pull it thru.

Reply:
@joseph-robertson I committed it here: https://github.com/NREL/OpenStudio-HPXML/commit/48a69e938a8289d53522969eceabb6e772a96817 after the fact because I forgot to initially. Should I revert the commit here and pull from OS-HPXML?

Reply:
No need to revert. We can just subtree pull on top of it.